install.py

```diff
@@ -0,0 +1,419 @@
+#!/usr/bin/env python
+
+# Copyright 2014 Andrea Micheli and Marco Gario
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import urllib2
+import os
+import tarfile
+import sys
+import platform
+import zipfile
+import argparse
+import sys
+import time
+
+#MG: TODO: Add credits for this class.
```

Comment:
Shall we add credits for this ProgressBar? Check license and maybe move to independent file.